Jenkinsfile-pr

```diff
@@ -24,6 +24,8 @@ pipeline {
         string(name: 'TEST_CASE', defaultValue: '*ST', description: 'maven parameter for executing specific tests')
         string(name: 'TEST_PROFILE', defaultValue: 'acceptance', description: 'maven parameter for executing specific test profile')
         string(name: 'EXCLUDE', defaultValue: 'networkpolicies', description: 'maven parameter for exclude specific test tag')
+        string(name: 'OPERATOR_IMAGE_PULL_POLICY', defaultValue: 'IfNotPresent', description: 'parameter for pulling operator images')
+        string(name: 'COMPONENTS_IMAGE_PULL_POLICY', defaultValue: 'IfNotPresent', description: 'parameter for pulling component images')
```

Comment:
It doesn't make sense to have it on both places - `environment` and `parameters.` I suggest keeping it only in `environment` part.